test/service/git/github/github-client.test.ts

```diff
@@ -38,4 +38,21 @@ describe("github service", () => {
     expect(res.commits).toEqual(["28f63db774185f4ec4b57cd9aaeb12dbfb4c9ecc"]);
   });
 
+  test("get pull request no squash: success", async () => {
+    const res: GitPullRequest = await gitClient.getPullRequest(TARGET_OWNER, REPO, MERGED_PR_FIXTURE.number, false);
```

Comment:
I would like to re-use an example of test that verifies the no-squahs behavior and get data from existing mocks for getPullRequest. But I did not find them It feels like I'm missing something here. Your guidance would be much appreciated.

Reply:
I think I have a lead around `test("multiple commits pr", async () => {`